Makefile

```diff
@@ -65,12 +66,14 @@ go-deps:
 	chmod +x ./vendor/k8s.io/code-generator/generate-internal-groups.sh
 
 SETUP_ENVTEST := $(shell command -v setup-envtest 2> /dev/null)
+SETUP_ENVTEST_VERSION := $(shell go list -m -json sigs.k8s.io/controller-runtime | jq -r '.Require[] | select(.Path == "sigs.k8s.io/controller-runtime/tools/setup-envtest") | .Version')
```

Comment:
suggestion: Try this instead:

```console
$ go list -m -json sigs.k8s.io/controller-runtime | jq -r '.Version'
```

Reply:
after dynamically setting the version in the Makefile from the contents of the go.mod file, i receive errors that the version of controller-runtime we are using does not include the setup-envtest package for v0.13.0.